src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -713,7 +717,7 @@ messageInstD ctxt parentIdent msgIdent messageParts =
             FieldNormal _fieldName fieldNum dpType options ->
                 do fieldE <- wrapE ctxt dpType options recordFieldName'
                    pure (apply encodeMessageFieldE [ fieldNumberE fieldNum, fieldE ])
-            FieldOneOf OneofField{subfields} ->
+            FieldOneOf (OneofField _ subfields) ->
```

Comment:
I prefer the original version because it works regardless of how many fields the `OneofField` constructor has, resulting in less code churn as people extend the data type

Reply:
I think I did this to fix some error, but it's unnecessary. Reverted